### DISCUSSION TITLE
Trilogy adapter

### DISCUSSION THREAD

**Discussion by zavan** [jeremyevans/sequel#2010] [General]
I'm using [trilogy](https://github.com/github/trilogy) as the MySQL adapter on a Rails app (via [activerecord-trilogy-adapter](https://github.com/github/activerecord-trilogy-adapter)).

Any plans for Sequel to support it? I can try implementing the adapter if so.

**Comment by jeremyevans** [comment]
At this point, it's probably best to have it as an external adapter.  If trilogy becomes much more popular, I could consider shipping support with Sequel.  I don't want to take that step now, because I don't think the additional maintenance burden is currently worth the benefits.

**Reply by jeremyevans** [reply]
Sequel 5.69.0 was released yesterday with a trilogy adapter.  What changed between March 27 and May 3 (when the trilogy adapter was committed) is that a trilogy adapter was committed to ActiveRecord on April 17, and I try to ensure that all adapters shipped with ActiveRecord have an equivalent in Sequel, to make it easier for users to transition from ActiveRecord to Sequel.
